pandas/core/generic.py

```diff
@@ -8623,6 +8623,13 @@ def tz_localize(self, tz, axis=0, level=None, copy=True,
         copy : boolean, default True
             Also make a copy of the underlying data
         ambiguous : 'infer', bool-ndarray, 'NaT', default 'raise'
+            When clocks moved backward due to DST, ambiguous times may arise.
+            E.g. in Central European Time (UTC+01), when going from 03:00 DST
+            to 02:00 non-DST, the naive time 02:30:00 occurs both at
+            00:30:00 UTC and at 01:30:00 UTC. In such a situation, the
+            `ambiguous` parameter can be used to correctly localize these
```

Comment:
I'd remove this last sentence. There's technically no _correct_ way to localize ambiguous times, pandas just provides the options below.

Reply:
I would actually argue that that is exactly what the `ambiguous` parameter does. There is no way to guess it correctly, that's for sure (that's what `ambiguous='infer'` would do). But if you have, say, 5 records of which you know the first 2 should be interpreted as DST and the other 3 as non-DST, you can use `ambiguous=np.array([True, True, False, False, False])` to do exactly that.
We encountered a situation where we had exactly this. There was a separate column indicating some sort of time zone ('CET' / 'CEST') but since 'CEST' is not an official timezone, we could not use that column as the timezone name to localize the times. However, we could use `ambiguous=df['timezone_col']=='CEST'`. In that case I think we were able to _correctly_ localize the times.

Reply:
Agreed. I guess I was getting caught up on the semantics. I would just rather say `the ambiguous parameter dictates how ambiguous times should be handled.`